app-editors/vis/vis-0.3.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```

Comment:
Since you are removing this version, there is no point in updating EAPI in it. The live ebuild is enough.